### DISCUSSION TITLE
Welcome to code.market Discussions!

### DISCUSSION THREAD

**Discussion by code-market-software** [code-market-software/code.market#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by liebensteyn** [comment]
pre-sale Question: I want to buy this https://code.market/product/flangapp-saas-online-app-builder-from-website but want to find out if ill get lifetime updates and a valid license. or is this a nulled version

**Reply by code-market-software** [reply]
Hello dear @liebensteyn. 
It's not a nulled product it's a regular licence. But this product published on our market via referral link. Author didn't published it directly on code.market.
